Resources/Locale/en-US/_Impstation/cosmiccult/cleansingsystem.ftl

```diff
@@ -0,0 +1,7 @@
+cleanse-deconvert-attempt-begin = {CAPITALIZE(THE($target))} holds a rosary out towards you!
+cleanse-deconvert-attempt-begin-user = You attempt to cleanse {CAPITALIZE(THE($target))} of otherworldly influences!
+cleanse-deconvert-attempt-success = You cleanse {CAPITALIZE(THE($target))} in a cascade of holy light!
+cleanse-deconvert-attempt-success-empowered = You cleanse {CAPITALIZE(THE($target))}, but a pulse of malign energy bursts forth!
```

Comment:
Not sure all of these need CAPITALIZE, grammatically, but I'm not exactly sure what $target is at this point